[blog] May 2019 Update (#16117)
mui-org/material-ui: pages/blog/may-2019-update.md

---
description: Here are the most significant changes in May.
---

# May 2019 Update

**Olivier Tassinari**

*June 8, 2019*

Here are the most significant changes in May:

> - 🎉 We have released Material-UI v4 stable during the React Europe conference.

Reviewer: ```suggestion
- 🎉 We released Material-UI v4 stable during the React Europe conference.
```

> You can find the details in our [blog post 📝](https://medium.com/material-ui/material-ui-v4-is-out-4b7587d1e701) and a quick introduction in this [5 minutes talk](https://www.youtube.com/watch?v=m54_CPfbWow).

Reviewer: Should the quick introduction come first, perhaps?

Author: There is a font size issue with the prez, but why not!

> - 🐛 We have made two consequent releases after v4.0.0, containing a significant amount of fixes (v4.0.1 and v4.0.2).

Reviewer: ```suggestion
- 🐛 We have made two subsequent releases after v4.0.0, containing a significant number of fixes (v4.0.1 and v4.0.2).
```

> But this summary is just scratching the surface. We have accepted 206 commits from 78 different contributors. We have changed 3,031 files with 96,616 additions and 85,768 deletions.

Reviewer: ```suggestion
But this summary is just scratching the surface. In the last two weeks, we have accepted 206 commits from 78 different contributors, changing 3,031 files with 96,616 additions and 85,768 deletions.
```

Author: four weeks?

## Our roadmap intent for June

*(We'll do our best, no guarantee!)*

- We will release an improved Slider component with range support. You can already [preview it](https://deploy-preview-15703--material-ui.netlify.com/components/slider/).

![Slider](/static/blog/may-2019-update/slider.png)

- We will work on a new Tree View component. You can already [preview it](https://deploy-preview-14827--material-ui.netlify.com/components/tree-view/).

![Tree View](/static/blog/may-2019-update/tree-view.png)

- We will add more themes to the [premium theme marketplace](https://themes.material-ui.com).
- We will start exploring how we can provide style adapters in v5 (styled-components, JSS, linaria, naked, etc.)
- We will experiment around advisory offerings for enterprises.
If you are an enterprise and are looking for one of the following:

>   - A dedicated communication channel. Have priority access to Material-UI engineers.

Reviewer: ```suggestion
  - A dedicated communication channel giving priority access to Material-UI engineers.
```

>   - Architecture Reviews. Sit down with a Material-UI engineer and walk through all aspects of your Material-UI project and current tech stack or architecture, to identify vulnerabilities and opportunities for improvement.

Reviewer: ```suggestion
  - Architecture Reviews. Sit down with a Material-UI engineer and walk through all aspects of your Material-UI project and current tech stack or architecture, to identify issues and opportunities for improvement.
```

---
description: Here are the most significant changes in May.
---

# May 2019 Update

**Olivier Tassinari**

*June 8, 2019*

Here are the most significant changes in May:

- 🎉 We have released Material-UI v4 stable during the React Europe conference.
You can find the details in our [blog post 📝](https://medium.com/material-ui/material-ui-v4-is-out-4b7587d1e701) and a quick introduction in this [5 minutes talk](https://www.youtube.com/watch?v=m54_CPfbWow).
- 🐛 We have made two consequent releases after v4.0.0, containing a significant amount of fixes (v4.0.1 and v4.0.2).

> But this summary is just scratching the surface. We have accepted 206 commits from 78 different contributors. We have changed 3,031 files with 96,616 additions and 85,768 deletions.

Reviewer 2: ❤️

## Our roadmap intent for June

*(We'll do our best, no guarantee!)*

- We will release an improved Slider component with range support. You can already [preview it](https://deploy-preview-15703--material-ui.netlify.com/components/slider/).

![Slider](/static/blog/may-2019-update/slider.png)

> - We will release a new Tree View component. You can already [preview it](https://deploy-preview-14827--material-ui.netlify.com/components/tree-view/).

Reviewer 2: The pressure! 😆

Author: I will help, we have 30 days :)

> (We'll do our best, no guarantee!)

Would you be more comfortable with "We will work on a new Tree View…?

Reviewer 2: Aiming for release in June might be a bit ambitious I'm all up for it but maybe `We will work on a new Tree View` is more realistic.

Reviewer 2: I'm thinking of moving the TreeView into the lab first. What do you think?

Author: Sounds good to me

![Tree View](/static/blog/may-2019-update/tree-view.png)

- We will add more themes to the [premium theme marketplace](https://themes.material-ui.com).
- We will start exploring how we can provide style adapters in v5 (styled-components, JSS, linaria, naked, etc.)
- We will experiment around advisory offerings for enterprises.
If you are an enterprise and are looking for one of the following:
  - A dedicated communication channel. Have priority access to Material-UI engineers.
  - Architecture Reviews. Sit down with a Material-UI engineer and walk through all aspects of your Material-UI project and current tech stack or architecture, to identify vulnerabilities and opportunities for improvement.

> You can contact us at `advisory at material-ui.com`.

Reviewer 3: ```suggestion
You can contact us at `advisory@material-ui.com`.
```
?

Author: Wanted to avoid bots, but maybe it doesn't matter.

Reviewer 3: Would be shocked if they are still that dumb

> - If we have the time, we will work on new components.

Reviewer 2: Is it worth adding this to the Roadmap? `If we have the time` isn't really mapping anything.
